assets/js/src/common/form/yesno/yesno.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { InputHTMLAttributes } from 'react';
+import classnames from 'classnames';
+
+type Props = InputHTMLAttributes<HTMLInputElement> & {
+  name: string,
+  onCheck: (isChecked: boolean) => void,
```

Comment:
Again, not sure if `onCheck` is a good name here.

Reply:
Same as for checkboxes and radios.